### PR TITLE
feature/account-page-add-sort-button

### LINKED ISSUE #29: Add button to sort students in table by name and surname

### PR DESCRIPTION
**Pull request by Ak1yamaKiyoshi**

### CODE REVIEW

File: src/components/pages/account-page/components/group-tab/GroupTab.tsx [hunk 1]
```diff
@@ -73,12 +81,25 @@ const GroupTab: FC = () => {
...
+        <Typography variant="h4">Список групи {user.group.code}</Typography>
+        <IconButton
+          // TODO move inline styles when refactor
+          sx={{ height: 'fit-content', marginLeft: '10px' }}
+          shape={IconButtonShape.SQUARE}
+          color={IconButtonColor.SECONDARY}
+          icon={isSorted ? <ChevronDownIcon /> : <ChevronUpIcon />}
+          onClick={() => {
+            setIsSorted(!isSorted);
+            refetch();
```
Comment (kujo205) [inline src/components/pages/account-page/components/group-tab/GroupTab.tsx:98]: Why do we need a `refetch` here?

File: src/components/pages/account-page/components/group-tab/GroupTab.tsx [hunk 2]
```diff
@@ -1,7 +1,14 @@
-import { FC } from 'react';
+import React, { FC, useState } from 'react';
 import { useQuery } from 'react-query';
-import { Box } from '@mui/material';
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/outline';
+import { Box, Typography } from '@mui/material';
 
+import Button from '@/components/common/ui/button-mui/Button';
```
Comment (kujo205) [inline src/components/pages/account-page/components/group-tab/GroupTab.tsx:6]:
```suggestion

```

File: src/components/pages/account-page/components/group-tab/GroupTab.tsx [hunk 3]
```diff
@@ -73,12 +81,25 @@ const GroupTab: FC = () => {
...
-    <div className={styles['content']}>
-      <div className={styles['text-content']}>
-        <h4>Список групи {user.group.code}</h4>
-      </div>
+    <Box className={styles['content']}>
+      <Box className={styles['text-content']}>
+        <Typography variant="h4">Список групи {user.group.code}</Typography>
+        <IconButton
+          // TODO move inline styles when refactor
+          sx={{ height: 'fit-content', marginLeft: '10px' }}
```
Comment (kujo205) [inline src/components/pages/account-page/components/group-tab/GroupTab.tsx:92]: Move them fr

**Review by kujo205 — CHANGES_REQUESTED** [review]

File: src/components/pages/account-page/components/group-tab/GroupTab.tsx (hunk 1 above)
Comment (Ak1yamaKiyoshi) [reply src/components/pages/account-page/components/group-tab/GroupTab.tsx:98]: Because otherwise student table will not update when sorting

**Pull request closed without merging**